files/en-us/web/javascript/reference/global_objects/object/isprototypeof/index.html

```diff
@@ -75,20 +75,23 @@ <h3 id="Using_isPrototypeOf">Using isPrototypeOf</h3>
 Bar.prototype = Object.create(Foo.prototype);
 Baz.prototype = Object.create(Bar.prototype);
 
-var baz = new Baz();
-
-console.log(Baz.prototype.isPrototypeOf(baz)); // true
-console.log(Bar.prototype.isPrototypeOf(baz)); // true
-console.log(Foo.prototype.isPrototypeOf(baz)); // true
+const foo = new Foo();
+const bar = new Bar();
+const baz = new Baz();
+
```

Comment:
```suggestion

// prototype chains:
// foo: Foo <- Object
// bar: Bar <- Foo <- Object
// baz: Baz <- Bar <- Foo <- Object
```

This is really a suggestion so up to you. But as I was reading the example and trying to work it out I found myself writing out the prototype chain for each object, and wondered if that might help other people?

Reply:
> This is really a suggestion so up to you. But as I was reading the example and trying to work it out I found myself writing out the prototype chain for each object, and wondered if that might help other people?

Yes, this is great — thanks. It succinct but adds a lot of clarity